### DISCUSSION TITLE
Roland MKS-30 CTRL panel

### DISCUSSION THREAD

**Discussion by esmeets79** [RomanKubiak/ctrlr#156] [General]
Hi I am looking for a CTRL panel or plugin to control my mks-30 with Hausensteiner mod. does anyone have this?

**Comment by bijlevel** [comment]
Did you look at: [https://ctrlr.org/panels/](url)?

**Reply by esmeets79** [reply]
Of course! it's not on there, that's why I'm asking here since the forum is offline ;)

**Reply by bijlevel** [reply]
I suspected it already, but I was like the support department" "Did you check if you computer is connected to the mains?" :-)

**Comment by dnaldoog** [comment]
@esmeets79 Maybe this panel could give you a starting point?

https://ctrlr.org/kiwi-3p-editor-update-version-3-0/

**Comment by dwmclean1** [comment]
I've made one that mostly works, except for saving and loading patches. If anyone wants to try finishing it off feel free.
[MKS-30.bpanelz.zip](https://github.com/RomanKubiak/ctrlr/files/6432408/MKS-30.bpanelz.zip)
